### DISCUSSION TITLE
Autosmooth for aquifers/damp stone

### DISCUSSION THREAD

**Discussion by Gix3612** [DFHack/dfhack#4158] [Ideas]
I'm still pretty new to Dwarf Fortress so this might not be as useful to others as I think it would, but, I would like to have a tool that will automatically smooth the damp stone in aquifers with high priority as they become exposed to help mitigate a lot of the micromanaging around dealing with them. This would especially be useful when combined with issue #1812.

**Comment by myk002** [comment]
that's not a bad idea. I'm working on an interface that will allow aquifer tiles to be dug without cancelling. an additional option for "smooth exposed aquifer tiles" would fit in with that
